src/pages/dashboard/workplace/components/Radar/index.tsx

```diff
@@ -0,0 +1,219 @@
+import { Axis, Chart, Coord, Geom, Tooltip } from 'bizcharts';
+import { Col, Row } from 'antd';
+import React, { Component } from 'react';
+
+import autoHeight from './autoHeight';
+import styles from './index.less';
+
+export interface RadarProps {
+  title?: React.ReactNode;
+  height?: number;
+  padding?: [number, number, number, number];
+  hasLegend?: boolean;
+  data: {
+    name: string;
+    label: string;
+    value: string | number;
+  }[];
+  colors?: string[];
+  animate?: boolean;
+  forceFit?: boolean;
+  tickCount?: number;
+  style?: React.CSSProperties;
+}
+interface RadarState {
+  legendData: {
+    checked: boolean;
+    name: string;
+    color: string;
+    percent: number;
+    value: string;
+  }[];
+}
+/* eslint react/no-danger:0 */
+class Radar extends Component<RadarProps, RadarState> {
+  state: RadarState = {
+    legendData: [],
+  };
+
+  chart: G2.Chart | undefined = undefined;
+
+  node: HTMLDivElement | undefined = undefined;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'node' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)